go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/google/go-querystring v1.1.0
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-retryablehttp v0.7.7
-	github.com/hashicorp/jsonapi v1.3.1
+	github.com/hashicorp/jsonapi v1.3.2-0.20240802183744-2490a9451c3d
```

Comment:
This is the [commit](https://github.com/hashicorp/jsonapi/commit/2490a9451c3dc98a9f3ae4f9e317a0b8646fa7c9) with the `Source` addition ErrorObject.

Will switch to the next released version whenever it happens, but this seems fine in the interim